### PR TITLE
added carousel code

### PR DESCRIPTION
**Pull request by salvik21**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,5 +17,16 @@
             <div class="logo-wrapper"><p class="logo">Renome</p></div>
         </nav>
     </header>
+    <section>
```
Comment (YuraLeo94) [inline index.html]: Please wrap it on html5 tag `<main>`
```
<main>
  <section>
    ...
  </section>
</main>
```

File: index.html [hunk 2]
```diff
@@ -16,5 +17,16 @@
...
+        <div class="carousel-wrapper">
```
Comment (YuraLeo94) [inline index.html]: It will be more sense if you call it just `class="carousel"`

File: index.html [hunk 3]
```diff
@@ -16,5 +17,16 @@
...
+            <div class="carousel__image-background"></div>
+            <div class="carousel__main-title"><h1>made with love</h1></div>
```
Comment (YuraLeo94) [inline index.html]: Rename please to `class="carousel__main-title-wrapper"`
Also better add a class name (`class="carousel__main-title"`) for `<h1>` since you add a specific style for it

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]
In the task design, we can see the image has a dark overlay. Please add it.
image
Please fix the issue what your image is bigger than `html, body`. Use dev tools(`F12`) to look at your dom elements
image

File: index.html [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="utf-8"/>
     <title>Renoma</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Abril+Fatface|Poppins">
```
Comment (YuraLeo94) [inline index.html]: By security fonts, images and etc. should be downloaded in your project and used from that.

File: index.html [hunk 5]
```diff
@@ -17,15 +17,17 @@
...
-            <div class="carousel__second-title"><h3>for you</h3></div>
-            <div class="carousel__forward-button"></div>
-            <div class="carousel__back-button"></div>
-        </div>
-    </section>
+    <main>
+        <section>
+            <div class="carousel">
+                <div class="carousel__image-background"></div>
+                <div class="carousel__main-title-wrapper"><h1 class="carousel__main-title">made with love</h1></div>
```
Comment (CoobQ) [inline index.html]: @salvik21 Please follow nesting approach
```
<div class="carousel__main-title-wrapper">
    <h1 class="carousel__main-title">made with love</h1>
</div>
```

File: style.css [hunk 6]
```diff
@@ -46,45 +52,51 @@ nav {
...
 section {
   clear: both;
+  height: auto;
+  width: 100%;
 }
 
 .carousel__image-background {
   position: absolute;
   margin-top: 15px;
+  filter: brightness(93%);
```
Comment (CoobQ) [inline style.css]: Please don't modify pictures color with css.  it can cause issue in the future if picture needs to be changed to something else and it can brake UI. Use original pictures

**Comment by CoobQ** [comment]
> In the task design, we can see the image has a dark overlay. Please add it. image

@YuraLeo94 Please don't modify pictures color with css. it can cause issue in the future if picture needs to be changed to something else and it can brake UI. Use original pictures

**Review by CoobQ — CHANGES_REQUESTED** [review]

File: style.css [hunk 7]
```diff
@@ -42,4 +48,75 @@ nav {
...
+.carousel__main-title-wrapper {
+  position: relative;
+  top: 240px;
+  left: 14px;
+}
+
+.carousel__second-title-wrapper {
+  position: relative;
+  top: 242px;
+  left: 14px;
```
Comment (YuraLeo94) [inline style.css]: Please don't use left 14px, since it will move the component with 100% width to the right which will make your app not 100% width, but 100%+14px which will create horizontal scroll (unacceptable for mobile app)

File: style.css [hunk 8]
```diff
@@ -42,4 +48,75 @@ nav {
...
+  font-size: 28px;
+  font-family: Poppins, sans-serif;
+  word-spacing: 7px;
+  letter-spacing: 3px;
+}
+
+.carousel__main-title-wrapper {
+  position: relative;
+  top: 240px;
+  left: 14px;
```
Comment (YuraLeo94) [inline style.css]: Please don't use left 14px, since it will move the component with 100% width to the right which will make your app not 100% width, but 100%+14px which will create horizontal scroll (unacceptable for mobile app)

File: style.css [hunk 9]
```diff
@@ -1,6 +1,12 @@
...
+* { 
+  margin: 0;
   padding: 0;
-  margin: 0;  
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  min-height: 100vh;
```
Comment (YuraLeo94) [inline style.css]: This won't be necessary when you fix the left position for the `title` and `second title`. Only need to add for body `margin: 0`

File: style.css [hunk 10]
```diff
@@ -42,4 +48,75 @@ nav {
   margin: 22px 0 0 5px;
   border-left: 1px solid #000000;
   height: 22px;
+}
+
+section {
+  clear: both;
```
Comment (YuraLeo94) [inline style.css]: Instead `clear: both;` please add `width` and `height` for `header` tag

File: style.css [hunk 11]
```diff
@@ -42,4 +48,75 @@ nav {
...
+  height: auto;
+  width: 100%;
+}
+
+.carousel__image-background {
+  position: absolute;
+  margin-top: 15px;
+  filter: brightness(93%);
+  width: 320px;
```
Comment (YuraLeo94) [inline style.css]: Please look at the `%` `width`, the picture should be full screen.

File: style.css [hunk 12]
```diff
@@ -42,4 +48,75 @@ nav {
...
+  height: 450px;
```
Comment (YuraLeo94) [inline style.css]: `height: 504px;` will look more by design

File: style.css [hunk 13]
```diff
@@ -1,6 +1,12 @@
-html, body {
+* { 
+  margin: 0;
   padding: 0;
-  margin: 0;  
+  box-sizing: border-box;
+}
```
Comment (YuraLeo94) [inline style.css]: It is bad practice, you should only add this for items that need it.

File: style.css [hunk 14]
```diff
@@ -42,4 +48,75 @@ nav {
   margin: 22px 0 0 5px;
   border-left: 1px solid #000000;
   height: 22px;
+}
+
+section {
+  clear: both;
+  height: auto;
+  width: 100%;
```
Comment (YuraLeo94) [inline style.css]: This `width` and `height` will be don't needed when you will fix others' comments, please remove them.

File: index.html [hunk 15]
```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="utf-8"/>
     <title>Renoma</title>
+    <link rel="stylesheet" href="fond/Poppins-SemiBold.ttf">
```
Comment (YuraLeo94) [inline index.html]: Looks like your fonts work well without this line, please remove it.

File: style.css [hunk 16]
```diff
@@ -42,4 +48,75 @@ nav {
...
+}
+
+.carousel__image-background {
+  position: absolute;
```
Comment (YuraLeo94) [inline style.css]: Are you sure you need it here(`position: absolute;`)?

File: style.css [hunk 17]
```diff
@@ -42,4 +48,75 @@ nav {
...
+  margin-top: 15px;
```
Comment (YuraLeo94) [inline style.css]: You need to add space between your `header` and `main` block. But you add space between your picture and the `carousel`. Why did you add this here?

File: style.css [hunk 18]
```diff
@@ -42,4 +48,75 @@ nav {
...
+.carousel__main-title {
+  color: #ffffff;
+  font-size: 28px;
+  font-family: Poppins, sans-serif;
+  word-spacing: 7px;
+  letter-spacing: 3px;
+}
+
+.carousel__main-title-wrapper {
+  position: relative;
```
Comment (YuraLeo94) [inline style.css]: Please read about the `position`

File: style.css [hunk 19]
```diff
@@ -42,4 +48,75 @@ nav {
...
+  top: 240px;
+  left: 14px;
+}
+
+.carousel__second-title-wrapper {
+  position: relative;
```
Comment (YuraLeo94) [inline style.css]: Please read about the `position`

File: style.css [hunk 20]
```diff
@@ -42,4 +48,75 @@ nav {
...
+.carousel__second-title {
+  color: #aa6d6d;
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  word-spacing: 7px;
+  letter-spacing: 3px;
+}
+
+.carousel__forward-button {
+  position: relative;
```
Comment (YuraLeo94) [inline style.css]: Please read about the `position`

File: style.css [hunk 21]
```diff
@@ -42,4 +48,75 @@ nav {
...
+  left: 90px;
+  width: 50px;
+  height: 50px;
+  background-image: url(images/forward.png);
+  background-size: cover;
+  border: 2px solid #fffffa;
+}
+
+.carousel__back-button {
+  position: relative;
```
Comment (YuraLeo94) [inline style.css]: Please read about the `position`

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]
Few links about the `position` in **CSS**
**eng**

- https://www.w3schools.com/css/css_positioning.asp

**ru**

- https://w3schoolsrus.github.io/css/css_positioning.html#gsc.tab=0
- https://html5book.ru/css-position/

File: style.css [hunk 22]
```diff
@@ -1,16 +1,23 @@
 * {
```
Comment (CoobQ) [inline style.css]: Please don't use "*" selector. If you want to make something on level higher then "body" use "html"
"*" - means everything

**Review by CoobQ — CHANGES_REQUESTED** [review]

**Review by CoobQ — DISMISSED** [review]

File: index.html [hunk 23]
```diff
@@ -9,12 +9,39 @@
...
+            </div>
+            <div class="logo-wrapper">
+                <p class="logo">Renome</p>
+            </div>
         </nav>
     </header>
+    <main>
+        <section>
+            <div class="carousel">
+                <div class="carousel__background-image-wrapper">
```
Comment (YuraLeo94) [inline index.html]: A word background in your class name makes me confused. Why is it needed? It will be made more sense for me if you call it `class="carousel__image-wrapper`

File: index.html [hunk 24]
```diff
@@ -9,12 +9,39 @@
...
+                    <img class="carousel__background-image" src="images/carousel_img_1.png" alt=" image_1">
```
Comment (YuraLeo94) [inline index.html]: Here same, just `class="carousel__image"`

File: style.css [hunk 25]
```diff
@@ -1,10 +1,15 @@
-html, body {
+body, html {
```
Comment (YuraLeo94) [inline style.css]: Why here need `html`?

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]
Please look at the **space between blocks** and fixe it (find a topic or an answer which describes this issue)
image

**Review by CoobQ — APPROVED** [review]

**Review by YuraLeo94 — APPROVED** [review]
